code/game/objects/items/devices/defibrillator.dm

```diff
@@ -268,3 +307,61 @@
 	item_state = "defib"
 	w_class = SIZE_SMALL
 	charge_cost = 99
+
+
+/obj/item/device/defibrillator/synthetic
+	name = "synthetic reset key"
+	desc = "Result of collaboration between Hyperdyne and Weyland-Yutani, this device can fix major glitches or programming errors of synthetic units, as well as being able to restart a synthetic that has suffered critical failure. It can only be used once before being reset."
+	icon = 'icons/obj/items/synth/synth_reset_key.dmi'
+	icon_state = "reset_key"
+	item_state = "defib"
+	w_class = SIZE_SMALL
+	charge_cost = 1000
```

Comment:
set force to 0 too . so the item doesnt get the "it would be a weak weapon"
it would also be neat if you can override the normal defib message of "it has 1/1 uses left" and "it can be used thru armor"